projects/metallb/metallb/helm/schema.json

```diff
@@ -2,188 +2,493 @@
   "$id": "https://packages.eks.amazonaws.com/metallb.schema.json",
```

Comment:
Shouldn't this change if we are using upstream schema?

Reply:
It's not exactly same, we do modify it a bit for packages so updated the id.